index.d.ts

```diff
@@ -75,6 +75,26 @@ declare const isStream: {
 	```
 	*/
 	transform(input: unknown): input is stream.Transform;
+
+	/**
+	@returns The [`type of stream`](https://nodejs.org/api/stream.html#stream_types_of_streams).
+
+	@example
+	```
+	import * as fs from 'fs';
+	import isStream = require('is-stream');
+
+	isStream.type(fs.createReadStream('unicorn.png'));
+	//=> readable
+
+	isStream.type(fs.createWriteStream('unicorn.png'));
+	//=> writable
+
+	isStream.type({});
+	//=> no-stream
+	```
+	*/
+	type(stream: unknown): stream is stream.Stream;
```

Comment:
The return type here is incorrect.

Reply:
Done 00f685f. However, I'm not sure if this was a good solution. It is the first time that I work with Typescript.